### PR TITLE
Ncgames 4

### PR DESCRIPTION
**Pull request by Greeny1248**
Finished NCgames Task 4

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -66,14 +66,13 @@ describe("app", () => {
...
-      test("returns in date descending order", () => {
+      test("returns in date descending order, this shows the oldest and newest film", () => {
         return request(app)
           .get("/api/reviews")
           .expect(200)
           .then((res) => {
-            console.log(res.body.reviews, "Created at desc order");
             expect(res.body.reviews[0].created_at).toBe(
-              "2021-01-18T10:01:41.251Z"
+              "2021-01-25T11:16:54.963Z"
```
Comment (MKD01) [inline __tests__/app.test.js:75]: This test can be done in a much simpler way using the jest-sorted library which i would recommend looking into.

File: db/test.sql [hunk 2]
```diff
@@ -6,4 +6,12 @@ SELECT  reviews.review_id, comments.review_id, owner, title, category, review_im
...
\ No newline at end of file
+ ORDER BY created_at DESC;
+
+ SELECT  reviews.review_id, reviews.owner, reviews.title, reviews.category, review_img_url, reviews.created_at, reviews.votes, reviews.designer,  COUNT (comments.review_id)::INT AS "comment_count"
+  FROM reviews
+  JOIN comments
+  ON reviews.review_id = comments.review_id
+  GROUP BY reviews.review_id
+  ORDER BY created_at DESC
+  ;`
```
Comment (MKD01) [inline db/test.sql:17]: Its absolutly fine to have a test.sql file here but i would recommend adding this file specifically to your git ignore file.

**Review by MKD01 — APPROVED** [review]
Happy for this to be merged after those changes have been made

File: db/test.sql (hunk 2 above)
Comment (Greeny1248) [reply db/test.sql:17]: Added to gitignore, thanks.

File: __tests__/app.test.js (hunk 1 above)
Comment (Greeny1248) [reply __tests__/app.test.js:75]: Completed with jest sorted

**Pull request merged**